ci-operator/step-registry/ipi/conf/nutanix/realtime/ipi-conf-nutanix-realtime-commands.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Create node MachineConfig files and set kernelType to realtime
+cat > "${SHARED_DIR}"/manifest_99-master-kerneltype.yaml <<EOF
```

Comment:
The same comment as above, I think we don't need it for masters.

Reply:
Fixed, thanks.